ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -1412,10 +1779,35 @@ private StoreConfig changeThreshold(int readOnlyThreshold, int readWriteDeltaThr
     return new StoreConfig(new VerifiableProperties(properties));
   }
 
-  private void restartStore(StoreConfig config, ReplicaId replicaId, ReplicaStatusDelegate delegate)
-      throws StoreException {
-    store.shutdown();
-    store = createBlobStore(replicaId, config, delegate);
-    store.start();
+  // ttlUpdateErrorCasesTest() helpers
+
+  /**
+   * Tests the case where the TTL update arrives when the the blob has not expired but cannot be TTL updated
+   * @throws Exception
+   */
+  private void inNoTtlUpdatePeriodTest() throws Exception {
```

Comment:
Minor: add the reason why TTL can't be updated in comment.

Reply:
Done